.github/workflows/publish-maven-snapshots.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish snapshots to maven
```

Comment:
I think this GA could be exluded. It should be in `main` only (parametrized with branch)

Reply:
This was added to self publish to maven so downstream branches would get changes more quickly [1] - which is needed to for the security build to get unblocked, is there another publishing process?

- [1] #6394

Reply:
Oh I see, it is a different GA (confused it with one we have)